resources/ory/charts/hydra/templates/deployment.yaml

```diff
@@ -2,9 +2,18 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: {{ include "hydra.fullname" . }}
+  {{- if .Release.Namespace }}
```

Comment:
why do we need that if statement here? is there a possibility that we won't have value of Release.Namespace?

Reply:
I don't think so. It's a copy from ORY chart, introduced by [this PR](https://github.com/ory/k8s/pull/82/files)